src/resmoke/resmoke_proxy.rs

```diff
@@ -0,0 +1,127 @@
+use std::{path::Path, str::FromStr, time::Instant};
```

Comment:
This code hasn't changed, it was just moved and then split out from the other resmoke suite configuration code.